docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -123,3 +123,26 @@ settings and filter the returned buckets based on a `min_doc_count` setting (by
 bucket that matches documents and the last one are returned). This histogram also supports the `extended_bounds`
 setting, which enables extending the bounds of the histogram beyond the data itself (to read more on why you'd want to
 do that please refer to the explanation <<search-aggregations-bucket-histogram-aggregation-extended-bounds,here>>).
+
+==== Missing value
```

Comment:
would this section not fit better in the general aggregations section since it affects (almost) every aggregation and is the same syntax for them all?